packages/varlet-ui/src/date-picker/DatePicker.vue

```diff
@@ -96,6 +96,17 @@
         />
       </transition>
     </div>
+    <div :class="n('date-action')" v-if="showAction">
+      <slot name="action">
+        <var-button text type="default" @click="onCancel">
+          {{ pack.pickerCancelButtonText }}
+        </var-button>
+
+        <var-button text type="primary" @click="onConfirm">
+          {{ pack.pickerConfirmButtonText }}
+        </var-button>
```

Comment:
The component's internationalized text should be independent and should not use picker definitions

Reply:
ok